### ISSUE TITLE
Vue.js files

### ISSUE THREAD

**Issue by xereda** [AtomLinter/linter-stylelint#403]
How do I apply stylelint in .vue files?

**Comment by ai** [comment]
Here is a PR https://github.com/AtomLinter/linter-stylelint/pull/406

**Comment by foundryspatial-duncan** [comment]
@xereda I don't think this issue should be closed until that pull request is merged, because the issue still exists in this repo, doesn't it? We can't make linter-stylelint use the right scopes to work in a vue component file.
